rpa-mapper/src/test/java/uk/gov/hmcts/cmc/rpa/mapper/RequestForJudgementJsonMapperTest.java

```diff
@@ -0,0 +1,135 @@
+    }
+
+    @Test
+    public void shouldMapRequestForJudgementPaidInFull() throws JSONException {
+        SampleCountyCourtJudgment countyCourtJudgment = new SampleCountyCourtJudgment();
+        countyCourtJudgment.withPaidAmount(PAID_ALREADY);
+        countyCourtJudgment.withPayBySetDate(PAY_BY_SET_DATE);
+        Claim claim = SampleClaim.builder()
+            .withCountyCourtJudgmentRequestedAt(CCJ_REQUESTED_AT)
+            .withCountyCourtJudgment(countyCourtJudgment.build()).build();
+        String expected = new ResourceReader().read("/judgement/rpa_request_for_judgement_full_by_set_date.json")
+            .trim();
+        assertEquals(expected, mapper.map(claim).toString(), STRICT);
+    }
+
+    @Test
+    public void shouldMapRequestForJudgementPayingFortnightly() throws JSONException {
+        RepaymentPlan repaymentPlan = SampleRepaymentPlan
+            .builder()
+            .withPaymentSchedule(PaymentSchedule.EVERY_TWO_WEEKS)
+            .withFirstPaymentDate(FIRST_PAYMENT_DATE)
+            .withInstalmentAmount(BigDecimal.valueOf(100.00)).build();
+        SampleCountyCourtJudgment countyCourtJudgment = new SampleCountyCourtJudgment();
+        countyCourtJudgment.withPaidAmount(PAID_ALREADY);
+        countyCourtJudgment.withRepaymentPlan(repaymentPlan);
+
+        Claim claim = SampleClaim.builder()
+            .withCountyCourtJudgmentRequestedAt(CCJ_REQUESTED_AT)
+            .withCountyCourtJudgment(countyCourtJudgment.build()).build();
+
+        String expected = new ResourceReader().read("/judgement/rpa_request_for_judgement_fortnightly.json")
+            .trim();
+        assertEquals(expected, mapper.map(claim).toString(), STRICT);
+    }
+
+    @Test
+    public void shouldMapRequestForJudgementPayingWeekly() throws JSONException {
+        RepaymentPlan repaymentPlan = SampleRepaymentPlan
+            .builder()
+            .withPaymentSchedule(PaymentSchedule.EACH_WEEK)
+            .withFirstPaymentDate(PAY_BY_SET_DATE).build();
+        SampleCountyCourtJudgment countyCourtJudgment = new SampleCountyCourtJudgment();
+        countyCourtJudgment.withPaidAmount(PAID_ALREADY);
+        countyCourtJudgment.withRepaymentPlan(repaymentPlan);
+
+        Claim claim = SampleClaim.builder().withDefendantEmail("defendant@email.com")
+            .withCountyCourtJudgmentRequestedAt(CCJ_REQUESTED_AT)
+            .withCountyCourtJudgment(countyCourtJudgment.build()).build();
+
+        String expected = new ResourceReader().read("/judgement/rpa_request_for_judgement_weekly.json")
+            .trim();
+        assertEquals(expected, mapper.map(claim).toString(), STRICT);
+    }
+
+    @Test
+    public void shouldMapRequestForJudgementPayingMonthly() throws JSONException {
+        SampleCountyCourtJudgment countyCourtJudgment = new SampleCountyCourtJudgment();
+        countyCourtJudgment.withPaidAmount(PAID_ALREADY);
+        SampleRepaymentPlan sampleRepaymentPlan = SampleRepaymentPlan
```

Comment:
Please avoid using word `sample` in variable names. In theory we could use it in every variable name used in tests but what would be the point?

Reply:
Correct. Removed!